src/igvfd/schemas/user.json

```diff
@@ -74,7 +74,7 @@
             "type": "array",
             "minItems": 1,
             "uniqueItems": true,
-            "permission": "import_items",
+            "permission": "admin_only",
```

Comment:
I noticed these were still import_items so I decided to change them on this ticket. 
Note that "import_items" and "admin_only" are arbitrary permission definitions that are not explicitly given to admins but instead admins just get all permissions so renaming it is only for clarity's-sake for non-admin users.